bin/paragon-scripts.js

```diff
@@ -13,12 +55,17 @@ const COMMANDS = {
 
   if (!executor) {
     // eslint-disable-next-line no-console
-    console.log(chalk.red.bold('Unknown command. Usage: paragon <command>'));
+    console.log(chalk.red.bold('Unknown command. Usage: paragon <command>.'));
+    return;
+  }
+
+  if (command === HELP_COMMAND) {
+    helpCommand(COMMANDS);
     return;
```

Comment:
I think you can avoid checking for this case if you define `help` command in `COMMANDS` objects as follows:

```js
help: {
  executor: () => helpCommand(COMMANDS),
  description: 'Displays help for available commands.',
},
```
this way try / catch block will work for `help` command as well